Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+    name: "BloomFilter",
+    platforms: [
+        .iOS("13.0"),
```

Comment:
Reckon we can bump this to iOS 14? Or is there a reason to keep it on 13?

Reply:
Can do